client/utils/helpers.js

```diff
@@ -0,0 +1,60 @@
+import jwtDecode from 'jwt-decode';
+
+export const summarize = (str, wordCount = 17) => str
+    .trim()
+    .replace(/\s+/, ' ')
+    .split(' ')
+    .splice(0, wordCount)
+    .join(' ')
+    .concat(' .....');
```

Comment:
This function can be simplified. You can achieve this functionality by doing this:

```javascript
str.trim().replace(/\s+/g, '  ').substr(0, wordCount).concat('...')
```